packages/font/src/google/retry.ts

```diff
@@ -0,0 +1,17 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import asyncRetry from 'async-retry'
+
+export async function retry<T>(
+  fn: asyncRetry.RetryFunction<T>,
+  retries: number
+) {
+  return await asyncRetry(fn, {
+    retries,
+    onRetry(e, attempt) {
+      console.error(
+        (e as Error).message + `\n\nRetrying ${attempt}/${retries}...`
+      )
+    },
+    minTimeout: 100,
```

Comment:
NOTE: The cooldown time for retries is based on the original code.
However, the second retry and later, the wait time doubles with each subsequent retry. This behavior is due to the default settings of async-retry and can be modified if needed. [async-retry API](https://github.com/vercel/async-retry#api)

Reply:
That's the default "Exponential Backoff" behavior of `async-retry` and IMHO we should keep this behavior.